language/module.md

```diff
@@ -0,0 +1,68 @@
+# Puppet Module Schema
+
+This document describes allowed (optional) and required files and directories in
+a Puppet Module.
+
+## What is a Puppet Module
+
+A module in the sense of this document is a compressed tar archive. It is
+usually distributed via [forge.puppet.com](https://forge.puppet.com/). A module
+is usually developed in a git repository. Some files in the git repository are
+only used for development and testing. They shall not be released.
+
+Common files often seen in a vcs repository that are used for development but
```

Comment:
"This is an incomplete list as this might vary on how tests and development are done."

Reply:
updated